e2e-tests/README.md

```diff
@@ -1,120 +1,54 @@
-
-Getting involved as a contributor
----------------------------------
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.stage.saucelabs.svg?label=stage)](https://webqa-ci.mozilla.com/job/socorro.stage.saucelabs/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.prod.saucelabs.svg?label=prod)](https://webqa-ci.mozilla.com/job/socorro.prod.saucelabs/)
 
-We love working with contributors to improve the Selenium test coverage for
-Socorro-Tests but it does require a few skills.  You will need to be familiar
-with Python, Selenium, and have a working knowledge of GitHub.
 
-If you are comfortable with Python, it's worth having a look at the Selenium
-framework to understand the basic concepts of browser-based testing and the
-page objects pattern.
+This directory holds Socorro client-based end-to-end tests which is why they're different than the rest of the code in this repository.
 
-If you need to brush up on programming but are eager to start contributing
-immediately, please consider helping out by doing manual testing.  You can
-help find bugs in Mozilla [Firefox][firefox] or find bugs in the Mozilla web
-sites tested by the [Web QA][webqa] team.  We have many projects that would be
-thrilled to have your help!
-
-To brush up on Python skills before engaging with us, [Dive Into Python][dive]
-is an excellent resource.  MIT also has [lecture notes on Python][mit] available
-through their open courseware.  The programming concepts you will need to know
-include functions, working with classes, and the basics of object-oriented
-programming.
+* PyPOM
+* pytest
+* pytest-selenium
+* pytest-xdist
+* requests
 
-Questions are always welcome
+Setup and run Socorro tests
 ----------------------------
-While we take great pains to keep our documentation updated, the best source of
-information is those of us who work on the project.  Don't be afraid to join us
-in irc.mozilla.org #mozwebqa to ask questions about our Selenium tests.  Mozilla
-also hosts the #breakpad chat room to answer your general questions about
-the Socorro project and about contributing to Mozilla.
-
-How to setup and run Socorro tests locally
----------------------------------------------
-This repository contains Selenium tests for the following environments:
-
-* development: https://crash-stats-dev.allizom.org
-* staging: https://crash-stats.allizom.org
-* production: https://crash-stats.mozilla.com
-
-Mozilla maintains a guide to running Automated tests on our QMO website:
-
-https://quality.mozilla.org/docs/webqa/running-webqa-automated-tests/
-
-This wiki page has some advanced instructions specific to Windows:
 
-https://wiki.mozilla.org/QA_SoftVision_Team/WebQA_Automation
+We suggest using a different virtual environment for these tests than the
+rest of Socorro so you're not mixing requirements:
 
+	$ mkvirtualenv socorro-tests
+	$ pip install -r requirements.txt
```

Comment:
You might want to include the `cd e2e-tests` to make sure the user is in the correct directory, or include the page in the commands.